books/architecture/src/rpc/handler/intro.md

```diff
@@ -1,2 +1,8 @@
 # The handler
-> TODO: fill after `cuprate-rpc-handler` is created.
\ No newline at end of file
+The handlers (functions that map requests into responses) are / can be generic with `cuprate-rpc-interface`.
+
+The main handlers used by Cuprate is implemented in `cuprated` itself, it implements the standard RPC handlers modeled after `monerod`.
+
+- JSON-RPC handlers: <LINK_TO_FILE>
```

Comment:
```suggestion
- JSON-RPC handlers: <LINK_TO_FILE>
```
lol

Reply:
These links do not exist yet, they depend on #355.